requirements/base.txt

```diff
@@ -58,3 +58,6 @@ dj-stripe==2.5.1
 
 # JWT used by Metabase embedding
 PyJWT==2.1.0
+
+# Used by the keyword/topic analyzer
+beautifulsoup4==4.11.1
```

Comment:
I think we're using https://github.com/rushter/selectolax in RTD, mostly for performance reasons. This is fine for now, but worth keeping an eye on if we're spending a lot of time parsing.